buildbot-conf/cordova.conf

```diff
@@ -327,6 +330,7 @@ c['schedulers'].extend([
             'cordova-wp8',
             'cordova-android-win',
             'cordova-blackberry-win',
+            'cordova-firefoxos',
```

Comment:
This is a list of builders that you define above, so you will want `cordova-firefoxos-win` and `cordova-firefoxos-osx` here.